wikipedia/wikipedia.py

```diff
@@ -89,13 +89,15 @@ def search(query, results=10, suggestion=False):
   * results - the maxmimum number of results returned
   * suggestion - if True, return results and suggestion (if any) in a tuple
   '''
-
+  offset = (page - 1) * results
```

Comment:
calculation of offset from page number. Page number seems to be more intuitive to me.